status/checkMissingPods_test.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2020 gRPC authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package status
+
+import (
+	grpcv1 "github.com/grpc/test-infra/api/v1"
+	"github.com/grpc/test-infra/config"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("CheckMissingPods", func() {
+
+	var currentLoadTest *grpcv1.LoadTest
+	var allRunningPods []*corev1.Pod
+	var actualReturn *LoadTestMissing
+	var expectedReturn *LoadTestMissing
+
+	BeforeEach(func() {
+		currentLoadTest = newLoadTestWithMultipleClientsAndServers()
+		allRunningPods = []*corev1.Pod{}
+		expectedReturn = &LoadTestMissing{Clients: []grpcv1.Client{}, Servers: []grpcv1.Server{}}
+	})
+
+	Describe("no pods from the current load test is running", func() {
```

Comment:
Generally, we use `Context` closures for inner-level descriptions, not `Describe`. When we describe something, we say "this is the thing I'm testing." Context says "this is the environment or context of the thing I'm testing."